test_runner/fixtures/neon_fixtures.py

```diff
@@ -1094,6 +1094,13 @@ def __init__(self, config: NeonEnvBuilder):
                 # Disable pageserver disk syncs in tests: when running tests concurrently, this avoids
                 # the pageserver taking a long time to start up due to syncfs flushing other tests' data
                 "no_sync": True,
+                # Batching (https://github.com/neondatabase/neon/issues/9377):
+                # enable batching by default in tests and benchmarks.
+                "page_service_pipelining": {
+                    "mode": "pipelined",
+                    "execution": "concurrent-futures",
+                    "max_batch_size": 32,
+                },
```

Comment:
Sigh, so this clever piece doesn't work because it breaks compatibility tests.

I guess I have to change the Pageserver's built in default then?

Reply:
found a way to make it work from Python without changing the Rust default